src/redux/store.js

```diff
@@ -0,0 +1,8 @@
+import { configureStore } from '@reduxjs/toolkit'
+import coderReducer from './coderSlice'
+
+export const store = configureStore({
+    reducer: {
+        coder: coderReducer,
+    },
+})
```

Comment:
missing a semi-colon. These sort of things can be prevented if we have a nice auto-formatting on save. Once we add a ticket as mentioned earlier for this, we can research what libraries or configurations to use to make this happen.